requirements.txt

```diff
@@ -21,3 +21,6 @@ sh==1.14.2
 semver==2.13.0
 pytest==7.1.1
 networkx==2.5.1
+insights-core>3.0.276
+git+https://gitlab.cee.redhat.com/ccx/ccx-ocp-core.git
```

Comment:
Does it work without VPN access?

Reply:
No, it's a very private repository that must not be available from outside

Reply:
So I think it's not fitting into this repo...
Unless somehow we make its installation optional in some way

Reply:
Agree, it needs to be optional

Reply:
Ok, extracted to a separate requirements file that will be handled by https://github.com/openshift/assisted-test-infra/pull/1712 inside our Jenkins